lib/jazzy/sourcekitten.rb

```diff
@@ -320,14 +320,18 @@ def self.deduplicate_declarations(declarations)
     def self.deduplication_key(decl)
       if decl.type.swift_extensible? || decl.type.swift_extension?
         [decl.usr, decl.name]
+      elsif decl.type.objc_class? || decl.type.objc_category?
+        name, = decl.objc_category_name || decl.name
```

Comment:
Stupid Rubocop. But now I see why you have objc_category_name returning two things!